guides/v3.5.0/tutorial/service.md

```diff
@@ -108,56 +61,59 @@ Let's get started creating our service by generating it through Ember CLI,
 which will create the service file, as well as a unit test for it.
 
 ```bash
-ember g service maps
+ember g service map-element
```

Comment:
Changed the service name to `map-element` because

* The new service I created that interacts with leaflet is called `map`
* This service actually creates an HTML element with a rendered map